app/views/teacher_training_adviser/steps/_identity.html.erb

```diff
@@ -6,7 +6,7 @@
 </p>
 
 <p>
-  To sign up, you'll need to have or be studying for a bachelor's degree, grade 2:2 or higher (or an equivalent).
+  To sign up, you'll need to have or be studying for a bachelor's degree with honours, grade 2:2 or higher (or an equivalent).
```

Comment:
@EllieNodder as before, I find this sentence a bit confusing as it is a few bits of information mushed together? Could it be split into two sentences?